apps/server/src/modules/learnroom/mapper/room-board-response.mapper.spec.ts

```diff
@@ -65,6 +65,33 @@ describe('room board response mapper', () => {
 			expect(result.elements[0] instanceof BoardElementResponse).toEqual(true);
 		});
 
+		it('should map tasks with status and its task card on board to response', () => {
+			const course = courseFactory.buildWithId();
+			const taskCard = taskCardFactory.buildWithId({ course });
+			const linkedTask = taskCard.task;
+			linkedTask.course = course;
+			linkedTask.taskCard = taskCard.id;
+			const status = {
+				graded: 0,
+				maxSubmissions: 0,
+				submitted: 0,
+				isDraft: false,
+				isFinished: false,
+				isSubstitutionTeacher: false,
+			};
+			const board = {
+				roomId: 'roomId',
+				displayColor: '#ACACAC',
+				title: 'boardTitle',
+				courseName: course.name,
```

Comment:
I am not sure why linter is not complaining, but the board params RoomBoardDTO does not contain courseName

Reply:
I just took that from the test above, but you are right. Maybe it was there before the refactoring to the SingleColumn. Will remove that from both tests.